examples/goext_example/example.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2017 NTT Innovation Institute, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/cloudwan/gohan/examples/goext_example/todo"
+	"github.com/cloudwan/gohan/extension/goext"
+)
+
+// Schemas returns a list of schemas that will be loaded with this extension
+func Schemas() []string {
+	return []string{
+		"todo/entry.yaml",
```

Comment:
Even if I deleted this line, I can build a plugin then pre_update extension worked. Why do we need to specify schema files here?